commands/root.go

```diff
@@ -89,6 +89,7 @@ func addCommands(cmd *cobra.Command, dockerCli command.Cli) {
 	)
 	if isExperimental() {
 		remote.AddControllerCommands(cmd, dockerCli)
+		addDebugShellCommand(cmd, dockerCli)
```

Comment:
Argh doesn't seem to work. @ktock I will take a look on my side and push related changes to your branch if you're ok?

Reply:
@crazy-max 

> I will take a look on my side and push related changes to your branch if you're ok?

Yes. Thank you for taking a look at this.

Reply:
Ok it seems that's because it's a new command and we don't have the page available on docs repo: https://github.com/docker/docs/tree/main/engine/reference/commandline

I will make some changes to the `docs-upstream` workflow to handle that. Keep you posted.

Reply:
@ktock Should be fixed with https://github.com/docker/buildx/pull/1651. Can you rebase?

Reply:
Passed CI, thanks!